go/src/koding/kites/kloud/main.go

```diff
@@ -55,6 +55,7 @@ func main() {
 	if err != nil {
 		log.Fatal(err)
 	}
+	defer k.Close()
```

Comment:
nope it doesnt see: 
https://github.com/koding/koding/blob/master/go/src/socialapi/workers/api/main.go#L102
and
https://github.com/koding/koding/blob/master/go/src/vendor/github.com/koding/runner/runner.go#L214-L233